fimax/loans/doctype/loan_application/loan_application.js

```diff
@@ -348,6 +351,7 @@ frappe.ui.form.on('Loan Application', {
 			return 0;
 		}
 
+		console.log(frm.fields_dict.loan_type.df.options, frm.doc.loan_type);
```

Comment:
Why do we need this line?

Reply:
Actually we don't, "Se Coló", we can remove it

Reply:
Great